ethernet_init/ethernet_init.c

```diff
@@ -15,7 +15,7 @@
 #if CONFIG_ETH_USE_SPI_ETHERNET
 #include "driver/spi_master.h"
 #endif // CONFIG_ETH_USE_SPI_ETHERNET
-#include "ethernet_init.h"
+#include "esp_eth_phy_lan867x.h"
```

Comment:
Consider adding to be conditionally compiled only 
```c
#if CONFIG_ETHERNET_PHY_LAN867X
```

Reply:
Updated.